ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -115,12 +133,18 @@ defaultVars() {
 }
 
 showArchList() {
-	echo "Currently supported architectures:
+	echo "Currently supported architectures and operating systems:
 	- ppc64le
+		- ubuntu18
```

Comment:
```suggestion
		- Ubuntu18
```

Reply:
Ive kept the os lower case here as this is the way we want users to input the parameter using the -o flag

Reply:
The case statement you put in for the OSs covers the capital first letter too. Given the distributions are names, it looks weird to me to not have them capitalised

Reply:
That sounds reasonable